SSR doesn't seem to work in development with Yarn

Hello! So far when attempting to follow the installation instructions, everything is working fine with the exception that I constantly get a flash of the Vue components not rendering before popping in when refreshing in development mode. I haven't tested SSR with the Node module yet. Using version `0.5.2`.

In the server logs, I see

```
[error] Vue SSR error: /Users/johngoff/Repos/Hipo/assets/node_modules/live_vue/priv/static/index.js:1 
export { getHooks } from "./hooks";                                                                   
^^^^^^                                                                                                
                                                                                                      
SyntaxError: Unexpected token 'export'                                                                
    at internalCompileFunction (node:internal/vm:76:18)                                                                                                                                                          at wrapSafe (node:internal/modules/cjs/loader:1283:20)        
    at Module._compile (node:internal/modules/cjs/loader:1328:27)         
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)                                                                                                                                         at Module.load (node:internal/modules/cjs/loader:1203:32)                                        
    at Module._load (node:internal/modules/cjs/loader:1019:12)            
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:203:29)                                                                                                                                     at ModuleJob.run (node:internal/modules/esm/module_job:195:25)                                   
    at async ModuleLoader.import (node:internal/modules/esm/loader:337:24)
    at async nodeImport (file:///Users/johngoff/Repos/Hipo/assets/node_modules/vite/dist/node/chunks/dep-BWSbWtLw.js:53056:15)
```

Which is repeated two or three times, seems like it's as many times as I have Vue components on the page. I think this may be related to https://github.com/Valian/live_vue/pull/32? But I'm not certain.

Interesting... I'm getting a different error:

```
[error] Vue SSR error: Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/jakub/Projects/live_vue_ssr_issue/assets/node_modules/live_vue/priv/static/hooks' imported from /Users/jakub/Projects/live_vue_ssr_issue/assets/node_modules/live_vue/priv/static/index.js
    at finalizeResolution (node:internal/modules/esm/resolve:257:11)
    at moduleResolve (node:internal/modules/esm/resolve:914:10)
    at defaultResolve (node:internal/modules/esm/resolve:1038:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:554:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:523:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:246:38)
    at ModuleJob._link (node:internal/modules/esm/module_job:126:49)
```

just I don't have a clue why installing through npm or yarn changes anything, after the install step it should get out of the way and let the node do it's job... I'm playing with it some more. Thanks for the repro!

--- a/assets/js/live_vue/app.ts
+++ b/assets/js/live_vue/app.ts
@@ -6,7 +6,7 @@ import type {
   LiveVueOptions,
   ComponentMap,
   LiveVueApp,
-} from "./types"
+} from "./types.js"
 
 /**
  * Initializes a Vue app with the given options and mounts it to the specified element.

--- a/assets/js/live_vue/hooks.ts
+++ b/assets/js/live_vue/hooks.ts
@@ -1,8 +1,8 @@
 import { createApp, createSSRApp, reactive, h, type App, SetupContext } from "vue"
-import { liveInjectKey } from "./use"
-import { mapValues } from "./utils"
-import { ComponentMap, LiveVue, LiveVueApp, LiveVueOptions } from "./types"
-import { migrateToLiveVueApp } from "./app"
+import { liveInjectKey } from "./use.js"
+import { mapValues } from "./utils.js"
+import { ComponentMap, LiveVue, LiveVueApp, LiveVueOptions } from "./types.js"
+import { migrateToLiveVueApp } from "./app.js"
 
 /**
  * Parses the JSON object from the element's attribute and returns them as an object.

--- a/assets/js/live_vue/index.ts
+++ b/assets/js/live_vue/index.ts
@@ -1,4 +1,4 @@
-export { getHooks } from "./hooks"
-export { useLiveVue } from "./use"
-export { createLiveVue } from "./app"
-export { findComponent } from "./utils"
+export { getHooks } from "./hooks.js"
+export { useLiveVue } from "./use.js"
+export { createLiveVue } from "./app.js"
+export { findComponent } from "./utils.js"

--- a/assets/js/live_vue/server.ts
+++ b/assets/js/live_vue/server.ts
@@ -2,9 +2,9 @@ import fs from "fs"
 import { basename, resolve } from "path"
 import { App, Component, createSSRApp, h } from "vue"
 import { renderToString, type SSRContext } from "vue/server-renderer"
-import { LiveVueApp, LiveHook } from "./types"
-import { migrateToLiveVueApp } from "./app"
-import { mapValues } from "./utils"
+import { LiveVueApp, LiveHook } from "./types.js"
+import { migrateToLiveVueApp } from "./app.js"
+import { mapValues } from "./utils.js"
 
 type Components = Record<string, Component>
 type Manifest = Record<string, string[]>

--- a/assets/js/live_vue/use.ts
+++ b/assets/js/live_vue/use.ts
@@ -1,5 +1,5 @@
 import { inject } from "vue"
-import type { LiveHook } from "./types"
+import type { LiveHook } from "./types.js"
 
 export const liveInjectKey = "_live_vue"
 

--- a/assets/js/live_vue/utils.ts
+++ b/assets/js/live_vue/utils.ts
@@ -1,4 +1,4 @@
-import { ComponentMap, ComponentOrComponentPromise } from "./types"
+import { ComponentMap, ComponentOrComponentPromise } from "./types.js"
 
 /**
  * Maps the values of an object using a callback function and returns a new object with the mapped values.

